fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java

```diff
@@ -130,6 +130,10 @@ public PlayerDto withStats(UnaryOperator<Character.BaseStatsDto> mod) {
         return withBaseStats(mod.apply(baseStats));
     }
 
+    public PlayerDto subtractStamina(int amount) {
```

Comment:
I implemented the same method in my PR… We should sync later on

Reply:
the person that comes first gets it merged, the other one has to fix the conflict